<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="FibHeapWidget" name="fibHeapView">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>56</y>
      <width>851</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>20</y>
      <width>983</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-99999999</number>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Insert">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_ExtractMin">
       <property name="text">
        <string>ExtractMin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_DecreaseKey">
       <property name="text">
        <string>DecreaseKey</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_SwitchHeap">
       <property name="text">
        <string>SwitchHeap</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_Union">
       <property name="text">
        <string>Union</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButton_NextStep">
       <property name="text">
        <string>NextStep</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_Clear">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_ClearAll">
       <property name="text">
        <string>ClearAll</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Selected_Heap"/>
    <addaction name="actionLoad_To_selected_Heap"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Selected_Heap">
   <property name="text">
    <string>Save Selected Heap</string>
   </property>
  </action>
  <action name="actionLoad_To_selected_Heap">
   <property name="text">
    <string>Load To selected Heap</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FibHeapWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">fibheapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
